run/test/specs/disappearing_message_checks_android.spec.ts

```diff
@@ -0,0 +1,477 @@
+  );
+  // Check if the 'Tap to download media' config appears
+  // Click on config
+  await device2.clickOnByAccessibilityID("Untrusted attachment message");
+  // Click on 'download'
+  await device2.clickOnByAccessibilityID("Download media");
+  // Wait for 10 seconds
+  await sleepFor(60000);
+  // Check if GIF has been deleted on both devices
+  await device1.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Media message",
+    maxWait: 1000,
+  });
+  await device2.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Media message",
+    maxWait: 1000,
+  });
+  await closeApp(device1, device2);
+}
+
+async function disappearingLinkMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const testLink = `https://type-level-typescript.com/objects-and-records`;
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+  await setDisappearingMessage(
+    platform,
+    device1,
+    ["1:1", "Disappear after send option", "30 seconds"],
+    device2
+  );
+  // await device1.navigateBack(platform);
+  // Send a link
+  await device1.inputText("accessibility id", "Message input box", testLink);
+  // Accept dialog for link preview
+  await device1.clickOnByAccessibilityID("Enable");
+  // No preview on first send
+  await device1.clickOnByAccessibilityID("Send message button");
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message sent status: Sent",
+    maxWait: 20000,
+  });
+  // Send again for image
+  await device1.inputText("accessibility id", "Message input box", testLink);
+  await sleepFor(100);
+  await device1.clickOnByAccessibilityID("Send message button");
+  // Make sure image preview is available in device 2
+  await device2.waitForTextElementToBePresent({
+    strategy: "id",
+    selector: "network.loki.messenger:id/linkPreviewView",
+  });
+  // Wait for 10 seconds to disappear
+  await sleepFor(10000);
+  await Promise.all([
+    device1.hasElementBeenDeletedNew({
+      strategy: "id",
+      selector: "network.loki.messenger:id/linkPreviewView",
+      maxWait: 1000,
+    }),
+    device2.hasElementBeenDeletedNew({
+      strategy: "id",
+      selector: "network.loki.messenger:id/linkPreviewView",
+      maxWait: 1000,
+    }),
+  ]);
+  await closeApp(device1, device2);
+}
+
+async function disappearingCommunityInviteMessage(
+  platform: SupportedPlatformsType
+) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const communityLink = `https://chat.lokinet.dev/testing-all-the-things?public_key=1d7e7f92b1ed3643855c98ecac02fc7274033a3467653f047d6e433540c03f17`;
```

Comment:
from constants.ts/testing.ts, search and replace to make sure there are no left over of this exept in one file